docs/source/getting-started/path-variables.rst

```diff
@@ -0,0 +1,118 @@
+.. Copyright (c) 2020 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
+.. SPDX-License-Identifier: Apache-2.0
+
+Setting JAVA_HOME and PATH variables
+####################################
+
+Windows
+*******
+We'll explain here how to set up ``JAVA_HOME`` and ``PATH`` variables on Windows.
+
+Setting the JAVA_HOME variable
+==============================
+
+1. Open ``Search`` and type "advanced system settings" and hit ``Enter``.
+2. Find the ``Advanced`` tab and click on the ``Environment Variables``.
+3. In the ``System variables`` section click on ``New`` if you want to set ``JAVA_HOME`` system wide. To set ``JAVA_HOME`` for a single user click on ``New`` under ``User variables``.
+4. In the opened modal window for ``Variable name`` type ``JAVA_HOME`` and for the ``Variable value`` set the path to the JDK installation. Click OK once you're done.
+5. Click OK and click Apply to apply the changes.
+
+Setting the PATH variable
+=========================
+If you have downloaded and installed the DAML SDK using our `Windows installer <https://github.com/digital-asset/daml/releases/latest>`_ your ``PATH`` variable is already set up.
+
+Mac OS
+******
+We'll explain here how to set up ``JAVA_HOME`` and ``PATH`` variables on Mac OS with ``bash`` console.
```

Comment:
macOS defaults to `zsh` as of Catalina. I suggest providing instructions for Catalina (and therefore `zsh`) first, and `bash` as an aside.

Reply:
Changed the OSX instructions to `zsh`